pages/[username]/respostas/[page]/index.public.js

```diff
@@ -0,0 +1,114 @@
+import { useRouter } from 'next/router';
+import { getStaticPropsRevalidate } from 'next-swr';
+
+import { ContentList, DefaultLayout, UserHeader } from '@/TabNewsUI';
+import { FaUser } from '@/TabNewsUI/icons';
+import { NotFoundError } from 'errors';
+import authorization from 'models/authorization.js';
+import content from 'models/content.js';
+import removeMarkdown from 'models/remove-markdown.js';
+import user from 'models/user.js';
+import validator from 'models/validator.js';
+import { useUser } from 'pages/interface';
+
+export default function Home({ contentListFound, pagination, username }) {
+  const { push } = useRouter();
+  const { user, isLoading } = useUser();
+  const isAuthenticatedUser = user && user.username === username;
+
+  return (
+    <DefaultLayout metadata={{ title: `Publicações · Página ${pagination.currentPage} · ${username}` }}>
+      <UserHeader username={username} respostasCount={pagination.totalRows} />
+
+      <ContentList
+        contentList={contentListFound}
+        pagination={pagination}
+        paginationBasePath={`/${username}/respostas`}
```

Comment:
Como estou sugerindo seguir o padrão de "responder" para o verbo e "comentário" para o substantivo, acho que podemos mudar na URL também.

```suggestion
        paginationBasePath={`/${username}/comentarios`}
```